pytools/graph.py

```diff
@@ -310,13 +331,13 @@ def compute_topological_order(graph: Mapping[T, Collection[T]],
 
 # {{{ compute transitive closure
 
-def compute_transitive_closure(graph: Mapping[T, MutableSet[T]]) -> (
-        Mapping[T, MutableSet[T]]):
+def compute_transitive_closure(graph: Mapping[NodeT, MutableSet[NodeT]]) -> (
```

Comment:
This doesn't work with `GraphT`? :cry:

Reply:
mypy doesn't like this: `pytools/graph.py:361: error: "Collection[Any]" has no attribute "add"  [attr-defined]` :-(